ideas/88-dapp-experience.md

```diff
@@ -0,0 +1,87 @@
+
+We had two brainstorming sessions during the Bangkok offsite. The notes from there are on the link below, please do not hesitate to comment:
+
+ https://docs.google.com/document/d/11MRsAO8GdIKVB0bk-osld-cYMMWDOVxB012QHPkQ6_k/edit?usp=sharing
+
+## Swarm Participants
+
+- Lead Contributor: @goranjovic
+- Testing & Evaluation: @asemiankevich 
+- Contributor: @lukaszfryc 
+- Contributor: <!-- @username -->
+- PM: @rachelhamlin
+- UX (if relevant): <!-- @username -->
+
+## Product Overview
+
+Objectives:
+
+1. Provide a definition of decentralized apps
+2. Provide clear inclusion criteria that dapps need to conform with in order to be listed in our catalog
+3. Perform some research o existing dapps for better understanding. For ex https://dappradar.com/
+4. Create the initial catalog 
+5. Provide a transparent and simple process for inclusion of new entries
+
+Key results:
+
+1. Users are able to easily use Status as a browser, having access to a nice and representative list of dapps.
+2. New users are introduced to the concept o dapps and why decentralization matters
+3. Developers who want to list their dapp in our catalog have a simple and fair process to do so and clear criteria their dapp needs to meet.
+
+Additional notes:
+
+- We may want to add some kind of user rating to allow users to help each other in dapp selection. This is completely independent from the review criteria that a dapp needs to meet in order to be included in the catalog. 
+- Inclusion criteria should not contain any subjective criteria such as UX or matters of taste. These concerns should be left to end users to decide.
+- We may use Status Open Bounty for the review process. The catalog could be a repository with a list of dapps, new entries could be submitted as pull requests and then reviewers would allow or decline based on the criteria. The exact incentivization strategy remains to be decided (e.g. dapp authors could stake some amount of money, reviewers would be rewarded, etc.)
+- We can use a separate repo for the catalog. In fact, DApp Catalog itself could be a dapp.
+
+
+
+### Product Description
+
+
+
+### Requirements & Dependencies
+
+
+### Minimum Viable Product
+
+MVP for this is a fixed initial DApp Catalog 
+
+Description: Currently we have 4 Ropsten dapp suggestions in the hardcoded list under Open DApp. This list should be expanded to include 12 DApps total.
+These dapps should be verified to work reasonably well (i.e. at all) and the selection should depend on currently selected network (only 
+Mainnet and Ropsten in scope for this MVP).
+
+## Dates
+Goal Date: 2018-04-18
+
+Description: Initial DApp Catalog
+
+Testing Days required: ?
+
+## Success Metrics
+
+- N dapps available out of the box in Status dapp catalog
+- X users opening a dapp from the catalog per day
+- Y users opening a dapp by URL
+- M new dapps submitted, reviewed and added to the catalog each month
+
```

Comment:
I like the success criteria. Can we use them to measure MVP?

Reply:
Actually, I think we should apply these criteria to the end of every iteration. E.g. after Iteration 0, the figures are like this, after we add new stuff in Iteration 1, the numbers go up so we must be doing something good, after Iteration 2 the numbers stay the same so that one is a flop, etc.

Reply:
It makes sense to use the exact same criteria since the ultimate goal of this swarm and of all its steps is the same - to increase awareness about dapps and subsequently also increase user interaction with them.

It should be interesting to see how those figures relate to each other. E.g. at some point we may (or may not) see that interaction with catalog dapps goes down and usage of custom urls goes up. E.g. as users become more interested in dapps and start looking for them beyond the "default" ones provided by Status.

Reply:
@lukaszfryc Do you have any suggestion on what we should measure?

Reply:
Maybe we should do some beta tests with selected people? Some way we did the UX tests for Onboarding. I'd like to test if they understand dapp concept, how to use dapps we offered them in Status and after some time - if they still use them and find them useful. Maybe @hesterbruikman have some thoughts about it?

Reply:
@goranjovic @lukaszfryc +1 for some observation. 

We also have a handy funnel set up from Create an account > Open DApp, which is hovering around 31% completion thus far. That'd be good to measure.